src/components/communications/Message/Message.tsx

```diff
@@ -64,9 +64,9 @@ export const Message: FunctionComponent<MessageProps> = ({
           <div className={className('content')}>{message.text}</div>
           {message.attachments.length > 0 && (
             <div className={`${className('content')} ${className('attachment')}`}>
-              {message.attachments.map((attachment) => (
-                <Attachment key={expand(attachment.resource).id} attachment={attachment} />
-              ))}
+              {message.attachments.map((attachment, index) => {
+                return <Attachment key={index} attachment={attachment} />
```

Comment:
LinkAttachments only have a `url` on them which is not guaranteed to be unique so I just went with index.